### PR TITLE
chore!: Normalize repository, dropping node <10.13 support

### PR DESCRIPTION
**Pull request by sttk**
This PR updates this project:

- Updates dependency versions
- Update GitHub Actions files
- Adds, changes and deletes some project files

Be careful that this package depends on [sparkle](https://github.com/gulpjs/sparkles) which will be bumped up.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,20 +19,27 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha --report lcovonly",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
     "sparkles": "^1.0.0"
```
Comment (phated) [inline package.json]: Needs the new release of `sparkles`. I'll work to get that release finished.

Comment (phated) [reply package.json]: @sttk sparkles v2.0.0 has been released. Please update it in this PR.

**Comment by sttk** [comment]
@phated I've updated sparkles to v2.0.0.

**Review by phated — APPROVED** [review]
Thanks @sttk!

**Pull request merged**